YoCode/Directory.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace YoCode
+{
+    public class Directory : IDirectory
+    {
+        public List<String> OriginalPaths { get; }
```

Comment:
I suggest next to the Lists of original and modified paths, we should have two strings for their respective root directories